russstation/code/modules/paperwork/pen.dm

```diff
@@ -0,0 +1,17 @@
+
+/obj/item/pen/laughter/attack(mob/living/M, mob/user)
+	if(!istype(M))
+		return
+
+	if(..())
+		if(reagents.total_volume)
+			if(M.reagents)
+
+				reagents.trans_to(M, reagents.total_volume, transfered_by = user, methods = INJECT)
+
+
+/obj/item/pen/laughter/Initialize()
+	. = ..()
+	create_reagents(45, OPENCONTAINER)
+	reagents.add_reagent(/datum/reagent/toxin/lexorin, 5)
```

Comment:
5u lexorin might be lethal, so be careful about that

Reply:
Yea, I used lexorin because it fits the idea of laughing so much you run out of breath, but if you know of another chem that's less lethal, I'm down to change it.

Reply:
since there's already a new reagent added, should it have a nerfed version of lexorin effects added to it? `adjustOxyLoss` and/or `losebreath` on a prob (maybe sharing the `prob(30)` for laughing)?

Reply:
Are you saying I should make the new reagent do oxyloss instead of having lexorin?

Reply:
yeah, if lexorin is too strong. i'm not sure how much is too much

Reply:
I'm not opposed to adding oxyloss stuff to super laughter reagent, but I need to get a better understanding of how the oxyloss stuff works first.

Reply:
To get this through, I'm going to call the lexorin a fine enough solution. We can change the super laughter variant in the future if the lexorin turns out to be too much.